h/static/scripts/components/GroupModeration.tsx

```diff
@@ -129,8 +131,15 @@ function AnnotationListContent({
         >
           <AnnotationCard
             annotation={anno}
-            onStatusChange={moderationStatus => {
-              onAnnotationStatusChangeStable(anno.id, moderationStatus);
+            loading={loading || moderationInProgress}
+            onStatusChange={async moderationStatusPromise => {
+              try {
```

Comment:
There is an implicit assumption here that the moderation status of one annotation cannot change while another changes. I would suggest adding a comment to explain this to a future reader.